internal/middleware/modelid_test.go

```diff
@@ -0,0 +1,166 @@
+package middleware
+
+import (
+	"context"
+	"testing"
+
+	"github.com/oklog/ulid/v2"
+	"github.com/openfga/openfga/pkg/storage/memory"
+	"github.com/openfga/openfga/pkg/typesystem"
+	"github.com/stretchr/testify/require"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"google.golang.org/grpc"
+)
+
+var ds = memory.New(10, 10)
+
+func TestUnaryModelIDInterceptor(t *testing.T) {
+	t.Run("empty_modelID_and_store_id_in_request", func(t *testing.T) {
+		interceptor := NewModelIDInterceptor(ds)
+
+		handler := func(ctx context.Context, req interface{}) (interface{}, error) {
+			return nil, nil
+		}
+
+		_, err := interceptor(context.Background(), nil, nil, handler)
+		require.NoError(t, err)
+	})
+
+	t.Run("no_modelID_in_request", func(t *testing.T) {
+		interceptor := NewModelIDInterceptor(ds)
+
+		handler := func(ctx context.Context, req interface{}) (interface{}, error) {
+			modelID, ok := ModelIDFromContext(ctx)
+			require.False(t, ok)
+			require.Empty(t, modelID)
+
+			return nil, nil
+		}
+
+		_, err := interceptor(context.Background(), &openfgapb.CheckRequest{StoreId: ulid.Make().String()}, nil, handler)
+		require.Error(t, err)
+	})
+
+	t.Run("empty_modelID_in_request_but_not_in_ds", func(t *testing.T) {
+		interceptor := NewModelIDInterceptor(ds)
+
+		handler := func(ctx context.Context, req interface{}) (interface{}, error) {
+			return nil, nil
+		}
+
+		_, err := interceptor(context.Background(), &openfgapb.CheckRequest{StoreId: ulid.Make().String()}, nil, handler)
+		require.Error(t, err)
```

Comment:
can you add a line like `require.Contains(err, "the expected error here")` so that the tests are easier to follow? i'm not sure what error is expected in this case, and `empty_modelID_in_request_but_not_in_ds` isn't helping me